WordPress/src/main/java/org/wordpress/android/ui/stats/service/StatsService.java

```diff
@@ -136,108 +135,150 @@ private void stopRefresh() {
                 }
             }
             mStatsNetworkRequests.clear();
-            mNumberOfFinishedNetworkCalls = 0;
-            mNumberOfNetworkCalls = 0;
         }
     }
 
-    private void startTasks(final String blogId, final StatsTimeframe timeframe, final String date,
-                            final boolean updateGraphStats, final boolean updateAlltimeStats, final int startId) {
-        this.mServiceBlogId = blogId;
-        this.mServiceRequestedTimeframe = timeframe;
-        this.mServiceRequestedDate = date;
-        this.mServiceStartId = startId;
+    private void startTasks(final String blogId, final StatsTimeframe timeframe, final String date, final StatsEndpointsEnum sectionToUpdate,
```

Comment:
No need for final parameters since the Thread has been removed (though it's not like it hurts).